dev-libs/paho-mqtt-c/paho-mqtt-c-1.3.12-r1.ebuild

```diff
@@ -59,9 +66,18 @@ src_test() {
 		return
 	fi
 
-	${EPYTHON} "${BUILD_DIR}"/test/"${MY_TEST_UTILS}"/interoperability/startbroker.py -c \
-			   "${BUILD_DIR}"/test/"${MY_TEST_UTILS}"/interoperability/localhost_testing.conf &
+	echo "RUNNING TESTS"
+	echo "Start test broker"
+	mv "${WORKDIR}"/"${MY_TEST_UTILS}"-"${MY_TEST_COMMIT}" "${WORKDIR}"/"${MY_TEST_UTILS}" || die
+	cd "${WORKDIR}"/"${MY_TEST_UTILS}/interoperability" || die
+	${EPYTHON} startbroker.py -c localhost_testing.conf &
+	sleep 2
```

Comment:
As discussed in IRC, using `sleep` to synchronize things is typically an anti-pattern.
Use
```bash
timeout 30 bash -c \
    'until printf "" 2>>/dev/null >>/dev/tcp/${0}/${1}; do sleep 1; done' \
    localhost "${PORT}" \
   || die "Timeout waiting for port ${PORT} to become available"
```
to wait 30 seconds for a port to become available.